### DISCUSSION TITLE
Accelerate the simulation speed of FlorisModel

### DISCUSSION THREAD

**Discussion by qdw319** [NatLabRockies/floris#1018] [General]
I tried to implement the UK Barrow offshore wind farm (30 turbines) using FlorisModel. For each time series data (wind speed and direction), the simulation time is around 0.3 second. Does anyone know how to accelerate the simulation speed of FlorisModel? for example, less than 0.1 second.

**Comment by misi9170** [comment]
Hi @Qdq315 ,

If you are running more than one condition in your time series data (for example, if you are running 100 time steps, which we refer to), then you may want to look into using the [`ParFlorisModel`](https://github.com/NREL/floris/blob/main/floris/par_floris_model.py). This will parallelize the computation across the conditions. 

You can find an example for the `ParFlorisModel` [here](https://github.com/NREL/floris/blob/main/examples/009_parallel_models.py), but it should work as a drop-in replacement for `FlorisModel`.

Alternatively, you could try using a different wake model, as some are simpler (and therefore faster to compute) than others, but there is of course a potential trade-off in model accuracy.

Best,
Misha
